Modules/Registration/PDEDeformable/wrapping/itkDemonsRegistrationFilter.wrap

```diff
@@ -1,5 +1,16 @@
 itk_wrap_class("itk::DemonsRegistrationFilter" POINTER)
-  foreach(s ${WRAP_ITK_SCALAR})
-    itk_wrap_image_filter_combinations("${s}" "${s}" "${WRAP_ITK_VECTOR_REAL}" 2+)
+  foreach(curr_dim ${ITK_WRAP_IMAGE_DIMS})
+    if(${curr_dim} LESS 1)
+        continue()
+    endif()
+    # Only vectors with 2 components are valid. Temporarily change the global vector dims list.
```

Comment:
Why only 2? It should work for many dimensions.

Reply:
Comment fixed here and other similar places.